src/main/java/com/hillel/items_exchange/dao/LocationRepository.java

```diff
@@ -0,0 +1,13 @@
+package com.hillel.items_exchange.dao;
+
+import com.hillel.items_exchange.model.Location;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.Optional;
+
+@Repository
+public interface LocationRepository extends JpaRepository<Location, Long> {
+
+    Optional<Location> findLocationByCityAndDistrict(String city, String district);
```

Comment:
As for me, it's better to rename this method to "findByCityAndDistrict(String city, String district)" because according to the "Spring Data JPA documentation" in the methods of JpaRepository not duplicate the name of returns entity (findById, findByName, etc).